scripts/installation/install-smg.sh

```
@@ -5,6 +5,8 @@ set -e
 # Usage: install-smg.sh [path-to-smg-src]
 # Default path: /tmp/smg-src
 
+export MAKEFLAGS="-j$(nproc)"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)